src/models/Settings.js

```diff
@@ -84,6 +84,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'features.consent': ['boolean', false, false],
       'features.idpDiscovery': ['boolean', false, false],
       'features.passwordlessAuth': ['boolean', false, false],
+      'features.showPasswordToggle': ['boolean', false, false],
```

Comment:
@rchild-okta FYI new feature been added. Also name seems too generic, but this is only for primary auth. Maybe `passwordToggleInPrimaryAuth`?  What do you think?

Reply:
discussed offline . Will change to `showPasswordToggleOnSignInPage`